study.md

```diff
@@ -152,7 +154,10 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+GET - tells sever to get information from URL
+POST - Similar to PUT, is used when "processing you wish to happen on the server side should be repeated, if the POST request is repeated", it is basically for on-purpose duplications
+PUT - allows you to add or create information in the resource identified by URL
+DELETE - this will delete the resource indentified by URL
 ```
```

Comment:
this is all the right idea but I wanted to point out that although the url is a big part of the request for something like delete, it's usually identifying the resource by an id, not the whole url itself. I think you have the right idea though. We'll be doing a lot with this soon and I think using it will make it clear.